lib/omniauth/strategies/bolt.rb

```diff
@@ -60,6 +60,9 @@ def callback_phase
         post_data = URI.encode_www_form(payload)
         res = http.request(req, post_data)
         response = JSON.parse(res.body)
+        @expiration_time = Time.now.utc + 15.minutes
```

Comment:
Why `15.minutes`? The expiration is defined by Bolt.
There is a `response['expires_in']` which defines the expiration time, it should be extended to the sandbox environment to 1 hour.
The expiration_time should be defined as `Time.current + response['expires_in'].seconds`

Reply:
fixed in https://github.com/nebulab/omniauth-bolt/pull/2/commits/56a86a997df1eccdacb8a76cf88d77482795dfcc